helpdesk_mgmt/models/res_users.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    helpdesk_team_ids = fields.Many2many(
```

Comment:
Why is this needed?

Reply:
This is necessary for the rule (`ir.rule`). Another possibility is to use `user_ids` field of tickets, which option do you think is better?

Reply:
We should balance performance and complexity. The question here is that this field is not computed, so how it's filled?

Reply:
This field comes from https://github.com/OCA/helpdesk/blob/13.0/helpdesk_mgmt/models/helpdesk_ticket_team.py#L12

Reply:
OK, please put a comment about that, and fix in both the name of the columns and table for avoiding surprises in the future.